cloudinit/userdata_cloudinit.erb

```diff
@@ -20,6 +20,13 @@ oses:
 hostname: <%= @host.shortname %>
 fqdn: <%= @host %>
 manage_etc_hosts: true
+ssh_pwauth: <%= @host.params['ssh_pwauth'] || true %>
+<% if @host.params['ssh_authorized_keys'] -%>
```

Comment:
I keep forgetting this, but if this is an empty string, will it still end up with a valid config or do you need to check if it's non-empty as well?